src/layouts/layout-container.hbs

```diff
@@ -11,6 +11,9 @@ layout: layout-base
         {{ else }}
           {{# withNewline }}{{ body }}{{/ withNewline }}
         {{/ inherit }}
+        {{# inherit stache.config.comments comments }}
```

Comment:
This is a boolean? If so, is it possible to use the boolean pattern from other boolean variables (ie, `showComments`)? If it's a pre-existing variable, don't worry about it.